python/paddle/fluid/layers/nn.py

```diff
@@ -6527,7 +6527,9 @@ def image_resize(input,
                  scale=None,
                  name=None,
                  resample='BILINEAR',
-                 actual_shape=None):
+                 actual_shape=None,
+                 align_corners=True,
+                 align_mode=1):
```

Comment:
Please make sure the default value compatible with older versions

Reply:
Confirmed, the default values are compatible